handbook/ops/bizops/customer_data_policy.md

```diff
@@ -1,15 +1,53 @@
 # Customer data policy
+
 How Sourcegraph handles customer information.
 
+Product data
+| Source/platform | Sourcegraph Cloud | Self-hosted | How long is this data kept for? | Who has access? |
+|------------------------------------------------------------------|--------------------|--------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|
+| Ping telemetry | | :heavy_check_mark: | Delete data for any instance that’s been offline for over 3 years. All active instance data are kept | All Sourcegraph teammates |
+| Google Cloud Platform data pipeline (Buckets, Dataflow, Pub/Sub) | :heavy_check_mark: | :heavy_check_mark: | Pub/Sub and Dataflow don’t hold data (only manage the data pipelines). GCP buckets follow the same rules as the data they backup (i.e. Sourcegraph.com data backed up in buckets is kept for a year) | Analytics and engineering teams |
+| Montoring and observabiltity tools (Sentry, Honeycomb) | :heavy_check_mark: | | 90 days (Sentry) and 60 days (Honeycomb) | Engineering teams |
+| Sourcegraph Cloud Postgres database | :heavy_check_mark: | | Retain indefinitely unless deletion request is received | Analytics and engineering teams |
+Sourcegraph.com user action/event logs | :heavy_check_mark: | | Retain data for 1 year | All Sourcegraph teammates |
+| Amplitude | :heavy_check_mark: | | Indefinitely | All Sourcegraph teammates |
+| Google Analytics | :heavy_check_mark: | | Indefinitely | All Sourcegraph teammates |
+
+User communications
```

Comment:
Under the User communications section, what does "Only instance admins and opted-in users" under the On-prem column mean in conjunction with the "All Sourcegraph teammates" under Who can access it? For the Cloud column, the check mark seems to say that all Sourcegraph teammates can access Sourcegraph Cloud data pulled into Slack--is that true? Then for an On-prem customer, can all Sourcegraph teammates access a customer's Slack support channels? I'm worried that an on-prem customer will review the customer data policy and think that we have a lot more access to their data than we might whenever they see "all Sourcegraph teammates"